SteamTogether.Scraper/Program.cs

```diff
@@ -38,6 +38,7 @@
             services.Configure<SteamOptions>(builder.Configuration.GetSection(SteamOptions.Steam));
 
             services.AddHttpClient();
+            // TODO(@ablizorukov): Make it non-transient?
```

Comment:
No, transient is fine, we need to have a new datatime each time we get the service

Reply:
`DateTimeService.UtcNow` already returns the current datetime on every access.